src/main/java/core/basesyntax/service/FruitOperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Inventory;
+import core.basesyntax.model.FruitTransaction;
+
+public interface FruitOperationHandler {
+    void getFruitOperation(FruitTransaction transaction, Inventory inventory);
```

Comment:
The method name `getFruitOperation` in the `FruitOperationHandler` interface suggests that it should return a fruit operation, but it's a void method. Consider renaming the method to reflect the action it performs, such as `handle` or `executeOperation`, to avoid confusion and improve readability in accordance with proper method naming conventions.